lib/src/test/java/space/taran/arklib/TestingUtils.kt

```diff
@@ -0,0 +1,13 @@
+package space.taran.arklib
```

Comment:
This class is a duplicate of `lib/src/androidTest/java/space/taran/arklib/TestingUtils.kt`.

Reply:
@kirillt we cannot access a file in androidTest in test and vice versa.
 If we want to have a single file then we can need to create a separate module testing utils and add it in build.gradle of lib for androidTestImplementation and testImplementation.

Reply:
We don't need instrumented tests for the moment. Only plain unit tests.

Reply:
@kirillt deleted the androidTest folder.